test/test-layout.js

```diff
@@ -0,0 +1,43 @@
+var should = require('should');
```

Comment:
So I vote don't change your quote style, @johnwobrien. It's in line with the style guide already.